pkg/config/config.go

```diff
@@ -730,3 +731,31 @@ func (lci *LocalConfigInfo) GetOSSourcePath() (path string, err error) {
 
 	return sourceOSPath, err
 }
+
+// IsValidArgumentList validates the argument list for `odo config` command
+func IsValidArgumentList(args []string) error {
+
+	if len(args) < 2 {
+		return fmt.Errorf("please provide a parameter name and value")
+	} else if len(args) > 2 {
+		return fmt.Errorf("only one value per parameter is allowed")
+	}
+
+	var err error
+	param, ok := asLocallySupportedParameter(args[0])
```

Comment:
Moving the enclosing function to the set command will require exporting this function.